docs/cluster-configuration.md

```diff
@@ -207,9 +207,11 @@ VMI example:
 **Default**: `false`
 
 ### enableManagedTenantQuota Feature Gate
-If set to true, enables the Managed Tenant Quota (MTQ) feature. See more details 
+If set to true, enables the Managed Tenant Quota (MTQ) feature. See more details
 [here](https://github.com/kubevirt/managed-tenant-quota).
```

Comment:
I think we can drop these two lines. Maybe also the "default" line - we don't care about it now, I think. 

Please just leave the line about deprecation.